network/src/commonMain/kotlin/com/wire/kalium/network/exceptions/KaliumException.kt

```diff
@@ -68,6 +70,10 @@ fun KaliumException.InvalidRequestError.isBadRequest(): Boolean {
     return errorResponse.label == BAD_REQUEST
 }
 
+fun KaliumException.InvalidRequestError.isNotFound(): Boolean {
+    return errorResponse.message == NOT_FOUND
```

Comment:
I just checked swagger and there is no message, so maybe we can change this to only check the code